lxd/db/node.go

```diff
@@ -463,6 +472,44 @@ func (c *ClusterTx) nodes(pending bool, where string, args ...interface{}) ([]No
 		return nil, err
 	}
 
+	rows.Close()
```

Comment:
This line should be a separate commit so it can be backported independently of the schema change feature.